src/system.c

```diff
@@ -84,13 +84,6 @@ Int enableCodeCoverageAtStartup( Char **argv, void * dummy);
 const Char * SyKernelVersion = "4.dev";
 
 /****************************************************************************
-*V  SyWindowsPath  . . . . . . . . . . . . . . . . . default path for Windows
-** do not edit the following line. Occurrences of `gap4dev'
-** will be replaced by string matching by distribution wrapping scripts.
-*/
-const Char * SyWindowsPath = "/cygdrive/c/gap4dev";
```

Comment:
Is it safe to remove this? Did you check that this really works on cygwin?

Reply:
Yes, I checked